mscore/capella.cpp

```diff
@@ -893,17 +893,15 @@ static Fraction readCapVoice(Score* score, CapVoice* cvoice, int staffIdx, const
 //TODO                        if (pm && (st == BarLineType::DOUBLE || st == BarLineType::END || st == BarLineType::BROKEN))
 //                              pm->setEndBarLineType(st, false, true);
 
-//TODO                        if (st == BarLineType::START_REPEAT || st == BarLineType::END_START_REPEAT) {
-                        if (st == BarLineType::START_REPEAT) {
+                        if (st == BarLineType::START_REPEAT || st == BarLineType::END_START_REPEAT) {
```

Comment:
Not sure how Capella represents this, but I'd propose we treat this the same way I am proposing treating it for the palette - just add the separate measure flags.